graphite/rules.go

```diff
@@ -27,6 +28,17 @@ import (
 	"github.com/criteo/graphite-remote-adapter/graphite/utils"
 )
 
+var (
```

Comment:
Maybe we should use https://github.com/patrickmn/go-cache to similar ?
(a string -> string thread-safe cache with supports for ttls)

Reply:
I will definitely try that!